src/java/us/kbase/templates/perl_client.vm.properties

```diff
@@ -115,67 +115,84 @@ sub new
     # We create an auth token, passing through the arguments that we were (hopefully) given.
 
     {
-	my %arg_hash2 = @args;
-	if (exists $arg_hash2{"token"}) {
-	    $self->{token} = $arg_hash2{"token"};
-	} elsif (exists $arg_hash2{"user_id"}) {
-	    my $token = Bio::KBase::AuthToken->new(@args);
-	    if (!$token->error_message) {
-	        $self->{token} = $token->token;
-	    }
-	}
-	
-	if (exists $self->{token})
-	{
-	    $self->{client}->{token} = $self->{token};
-	}
+        my %arg_hash2 = @args;
+        if ( exists $arg_hash2{ token } ) {
+            $self->{ token } = $arg_hash2{ token };
+        }
+        elsif ( exists $arg_hash2{ user_id } ) {
+            my $token = Bio::KBase::AuthToken->new( @args );
+            if ( !$token->error_message ) {
+                $self->{ token } = $token->token;
+            }
+        }
+
+        if ( exists $self->{ token } ) {
+            $self->{ client }{ token } = $self->{ token };
+        }
     }
 #end
 
-    my $ua = $self->{client}->ua;	 
-    my $timeout = $ENV${empty_escaper}{CDMI_TIMEOUT} || (30 * 60);	 
-    $ua->timeout($timeout);
+    my $ua      = $self->{ client }->ua;
+    my $timeout = $ENV${empty_escaper}{ CDMI_TIMEOUT } || ( 30 * 60 );
+    $ua->timeout( $timeout );
     bless $self, $class;
-    #    $self->_validate_version();
     return $self;
 }
 
 #foreach( $module in $modules )
 #if( $any_async || $async_version)
-sub _check_job {
-    my($self, @args) = @_;
 # Authentication: ${method.authentication}
```

Comment:
moved this line out of the method and into the header

Reply:
prior to change:
```
sub _check_job {
    my ($self, @args) = @_;
    # Authentication: ...
```

after change:
```
# Authentication
sub _check_job {
    my ($self, @args) = @_;
```

i.e. the comment isn't stuck in the middle of the method